list.json

```diff
@@ -499,13 +499,6 @@
     "language": "Fortran",
     "licence": "GPL-3.0"
   },
-  {
-    "name": "Firedrop2",
-    "website": "http://www.voidspace.org.uk/python/firedrop2/",
```

Comment:
Current website doesn't seem to exist anymore, causing the tests to fail.

Reply:
But why would you remove the whole SSG then?

Reply:
Seems like either website or repository link are required.

If I only remove the website property error below is thrown:

```
Comparison Error:
AssertionError [ERR_ASSERTION]: missing required fields on Firedrop2
    at equal (/Users/manvelsaroyan/projects/static-gen-list/master/node_modules/assert-helpers/source/index.js:171:10)
    at /Users/manvelsaroyan/projects/static-gen-list/master/source/test.js:59:4
    at Array.forEach (<anonymous>)
    at EventEmitterGrouped.<anonymous> (/Users/manvelsaroyan/projects/static-gen-list/master/source/test.js:56:8)
    at ambi (/Users/manvelsaroyan/projects/static-gen-list/master/node_modules/ambi/source/index.js:91:18)
    at Domain.fireMethod (/Users/manvelsaroyan/projects/static-gen-list/master/node_modules/taskgroup/source/lib/task.js:519:5)
    at Domain.run (domain.js:242:14)
    at EventEmitterGrouped.fire (/Users/manvelsaroyan/projects/static-gen-list/master/node_modules/taskgroup/source/lib/task.js:537:15)
    at Immediate.queue (/Users/manvelsaroyan/projects/static-gen-list/master/node_modules/taskgroup/source/lib/task.js:572:20)
    at runCallback (timers.js:810:20)
```

If I keep the non existing website test are failing:

```
Error #1:
static site generators list ➞ uris are valid / still exist ➞ Firedrop2: http get website: http://www.voidspace.org.uk/python/firedrop2/
AssertionError [ERR_ASSERTION]: response http status code should be 200 success
    at equal (/Users/manvelsaroyan/projects/static-gen-list/master/node_modules/assert-helpers/source/index.js:171:10)
    at /Users/manvelsaroyan/projects/static-gen-list/master/source/test.js:27:4
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
```

Googling didn't help me to find any existing repository associated with current project.

Having unmaintained list item causing the list being outdated, sounded to me something that's easier to remove.